### DISCUSSION TITLE
any updates on buttons and prints being fixed

### DISCUSSION THREAD

**Discussion by Vertigo1207** [NoMeansNowastaken/KarutaSniper#16] [General]
.

**Comment by splat58** [comment]
when i get back from vacation
(this is my laptop acc and nomeansno is my desktop)

**Reply by Vertigo1207** [reply]
oh that makes sense now aha

**Comment by Vertigo1207** [comment]
i dont know how this would work but any way you could teach me how the script works

**Reply by splat58** [reply]
the main part of the script works like this:
1. listen for messages
2. check that a) karuta bot sent the message and b) it is a channel in config.json
3. check that the message is a card drop message
4. use ocr (pytesseract) to read the info needed on the cards
5. check if the text matches one of the categories
6. check if its a button and if so, click on it (in theory)
7. if not a button, react to it
